source/reference/database-references.txt

```diff
@@ -227,6 +227,12 @@ Driver Support for DBRefs
 
      - You can traverse references manually.
 
+   * - Go
```

Comment:
Should these be in alphabetical order? If so, this should be above the "Haskell" line.

Reply:
Oh my mistake you're right! Will fix. Cheers.